layouts/partials/footer.hbs

```diff
@@ -1,6 +1,6 @@
 <a href="#" id="scroll-to-top">&uarr; <span>{{site.scrollToTop}}</span></a>
 
-<footer{{#if className}} class="{{className}}"{{/if}}>
+<footer {{#if className}} class="{{className}}" {{/if}}>
```

Comment:
As you added `id-class-ignore-regex`, the latter change isn't needed. But it's your choice.
```suggestion
<footer {{#if className}} class="{{className}}"{{/if}}>
```

Reply:
>In my environment, npm run test runs htmllint against only 18 files. Can you also correct the script?

I ran on window 10 and I didn't see this mistake……Well, I changed according to your ideas, maybe you can fetch the latest version of codes and have a try;)

Reply:
Yes, maybe wildcard is treated differently based on OSes. Also, other two files can be updated, too!